numalogic/connectors/druid/_druid.py

```diff
@@ -123,9 +125,8 @@ def fetch(
         )
         try:
             response = self.client.groupby(**query_params)
-        except Exception:
-            _LOGGER.exception("Problem with getting response from client")
-            return pd.DataFrame()
+        except Exception as err:
+            raise DruidFetcherError("Druid Exception:\n") from err
```

Comment:
Can you also add in the docstring that it raises this error?

Reply:
Good point!